src/inmanta/plugins.py

```diff
@@ -421,6 +426,11 @@ def __call__(self, *args: object, **kwargs: object) -> object:
         The function call itself
         """
         self.check_requirements()
+        if self.deprecated:
+            msg: str = f"Plugin '{self.__function_name__}' in module '{self.__module__}' is deprecated. "
+            if self.replaced_by:
+                msg += f"It should be replaced by '{self.replaced_by}'"
```

Comment:
```suggestion
            msg: str = f"Plugin '{self.__function_name__}' in module '{self.__module__}' is deprecated."
            if self.replaced_by:
                msg += f" It should be replaced by '{self.replaced_by}'"
```
No trailing space if `replaced_by` is not defined.